content/_data/changelogs/weekly.yml

```diff
@@ -2520,6 +2520,55 @@
     # pull: 3395 too minor (French translation)
     # pull: 3355 too minor (Chinese translation)
     # pull: 3396 too minor (f:textbox checkMethod)
+- version: "2.119"
+  date: 2018-04-25
+  changes:
+    - type: rfe
+      issue: 31661
+      pull: 3082
+      message: >
+        Ensure as much as possible that the Jenkins root URL is defined by adding a new setup wizard page and an administrative monitor.
+    - type: rfe
+      pull: 3082
+      message: >
+        Developer: Add a new overload for <code>HttpResponses#errorJSON</code>.
```

Comment:
Move it to the bottom? I also can barely understand what it means, and the PR is huge. Some extra words would help